packages/material-ui/src/Box/Box.js

```diff
@@ -29,9 +31,143 @@ export const styleFunction = css(
   ),
 );
 
+function omit(input, fields) {
+  const output = {};
+
+  Object.keys(input).forEach((prop) => {
+    if (fields.indexOf(prop) === -1) {
+      output[prop] = input[prop];
+    }
+  });
+
+  return output;
+}
+
+const boxProps = [
```

Comment:
We could potentially use https://github.com/mui-org/material-ui/blob/876207a131ab12d23bcc108941647d9dd54eb793/packages/material-ui-styles/src/styled/styled.js#L62 instead